ros_buildfarm/templates/__init__.py

```diff
@@ -72,14 +75,18 @@ def _escape_value(value):
 
 
 def _expand_snippet(snippet_name, **kwargs):
+    """Expand snippet using given snippet_name"""
     template_name = 'snippet/%s.xml.em' % snippet_name
     _expand_template(template_name, **kwargs)
 
 
 def _expand_template(template_name, **kwargs):
-    global template_basepath
+    """Expand template call for interpreter"""
+    template_name = os.path.join('templates', template_name)
+    template_package = _find_first_template(template_name, **kwargs)
+
     global interpreter
-    template_path = os.path.join(template_basepath, template_name)
+    template_path = pkg_resources.resource_filename(template_package, template_name)
```

Comment:
You can still use this if you add the package source dir to the python package path.
This is how I am able to code, run, debug the package from the source dir in my git repo.
I think if one would want to use a python package, they'd likely add it to the PYTHONPATH, but I'm not sure how else this being package might be used.